source/features/cross-deleted-pr-branches.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {wrap} from '../libs/dom-utils';
+
+function init(): void {
+	const lastBranchAction = select.last('.TimelineItem-body .user-select-contain > span:not(.base-ref)');
+	if (!lastBranchAction) {
+		return;
+	}
+
+	if (!lastBranchAction.closest('.TimelineItem-body')!.textContent!.includes(' deleted ')) {
+		return;
+	}
+
+	const deletedBranchName = lastBranchAction.textContent!.trim();
+	const repoRoot = (select('.head-ref a')! as HTMLAnchorElement).href.replace(/\/tree.*$/, '');
```

Comment:
The regex would not work if the repo or username starts with `tree`. Use this:

```suggestion
	const repoRoot = select<HTMLAnchorElement>('.head-ref a')!.href.split('/', 5).join('/')
```